webapp/client/src/pages/SubmitAcknowledgePage.js

```diff
@@ -17,11 +17,12 @@ export default function SubmitAcknowledgePage({ prevUrl, logoutUrl }) {
     <>
       <StepHeaderButtons url={prevUrl} />
       <FormSuccessHeader {...stepHeader} />
-      <h2 className="h4 mt-4">{t("submitAcknowledge.next-steps.heading")}</h2>
-      <p>{t("submitAcknowledge.next-steps.text")}</p>
+      <h2 className="h4 mt-5">{t("submitAcknowledge.next-steps.heading")}</h2>
```

Comment:
Why does this heading get a different margin-top? Is that ✅ with design?

Reply:
This is probably the spacing between the blocks, right?

Reply:
It was a quick fix, because I was setting also a mb for the p elements and the first h2 doesn't have any p before. I just changed that to:

`            <h2 className="h4 mt-5">{t("submitAcknowledge.next-steps.heading")}</h2>
            <p>{t("submitAcknowledge.next-steps.text")}</p>
            <h2 className="h4 mt-5">{t("submitAcknowledge.recommend.heading")}</h2>
            <p>{t("submitAcknowledge.recommend.text")}</p>
            <h2 className="h4 mt-5">{t("submitAcknowledge.logout.heading")}</h2>
            <p className="mb-5">{t("submitAcknowledge.logout.text")}</p>`

This way we only set spacing using the h2 and for the button to have also some spacing from the last p, I set the mb in the las tp. wdyt?

Reply:
👍 That's fine with me as long as it is fine with design.